.github/workflows/velox_backend.yml

```diff
@@ -56,19 +56,29 @@ jobs:
   build-native-lib-centos-7:
     runs-on: ubuntu-20.04
     container: apache/gluten:vcpkg-centos-7
+    env:
+      VELOX_BUILD_PATH: "./ep/build-velox/build/velox_ep/_build/release/"
     steps:
       - uses: actions/checkout@v2
       - name: Generate cache key
         run: |
-          echo ${{ hashFiles('./ep/build-velox/src/**', './dev/**', './cpp/*', './.github/workflows/*') }} > cache-key
+          echo ${{ hashFiles('./ep/build-velox/src/**', './dev/**', './.github/workflows/velox_backend.yml') }} > cache-key
```

Comment:
> ./ep/build-velox/src/**

I think we could change our OAP Velox dependency from branches to tags while landing this PR. Branches can be updated without invalidating the new cache.

Reply:
@zhztheplayer, yes, this is a potential issue. But before a velox rebase branch is finalized, we may still need oap/velox to hold it as a branch that allows updating (e.g., needs some fixes due to Gluten CI failure). So maybe, we should just have a internal convention to forbid updating oap/velox branch once it is already referenced by Gluten code.